docs/cep/query-guide/named-aggregation.md

```diff
@@ -61,38 +61,37 @@ the aggregation input stream should only feed events to one aggregation definiti
 
 **Example**
 
-This Stream Application defines an aggregation named `TradeAggregation` to calculate the average and sum for the `price` attribute of events arriving at the `TradeStream` stream. These aggregates are calculated per every time granularity in the second-year range.
+This Stream worker defines an aggregation named `TradeAggregation` to calculate the average and sum for the `price` attribute of events arriving at the `TradeStream` stream. These aggregates are calculated per every time granularity in the second-year range.
```

Comment:
```suggestion
This stream worker defines an aggregation named `TradeAggregation` to calculate the average and sum for the `price` attribute of events arriving at the `TradeStream` stream. These aggregates are calculated per every time granularity in the second-year range.
```

Reply:
The second sentence of this makes no sense.

Reply:
It means that the aggregations will be calculated per whatever time granularity you have specified (whether "sec" or "min" or "hour" or "day" or "month" or "year") These six values are acceptable and that is what it means by sec-year range it seems.